### PR TITLE
Add linter checks and fix linter warnings

### LINKED ISSUE #24: Add linter to inx-participation

### PR DESCRIPTION
**Pull request by muXxer**

### CODE REVIEW

File: pkg/participation/participation_manager.go [hunk 1]
```diff
@@ -165,51 +166,59 @@ func (pm *ParticipationManager) CloseDatabase() error {
...
+func (pm *Manager) eventIDsWithoutLocking(eventPayloadType ...uint32) []EventID {
 	events := pm.events
 	if len(eventPayloadType) > 0 {
 		events = filteredEvents(events, eventPayloadType)
 	}
 
-	var ids []EventID
+	ids := make([]EventID, len(events))
+	i := 0
 	for id := range events {
```
Comment (alexsporn) [inline pkg/participation/participation_manager.go:192]: huh? why not iterate with index and value?

File: pkg/participation/participation_manager_test.go [hunk 2]
```diff
@@ -9,7 +10,7 @@ import (
 	"github.com/iotaledger/hive.go/serializer/v2"
 	"github.com/iotaledger/hornet/v2/pkg/model/storage"
 	"github.com/iotaledger/inx-participation/pkg/participation"
-	"github.com/iotaledger/inx-participation/pkg/participation/test"
+	participation_test "github.com/iotaledger/inx-participation/pkg/participation/test"
```
Comment (alexsporn) [inline pkg/participation/participation_manager_test.go]: this is weird, that is called the same as the current package

File: pkg/participation/staking_rewards_test.go [hunk 3]
```diff
@@ -6,7 +7,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/iotaledger/inx-participation/pkg/participation"
-	"github.com/iotaledger/inx-participation/pkg/participation/test"
+	participation_test "github.com/iotaledger/inx-participation/pkg/participation/test"
```
Comment (alexsporn) [inline pkg/participation/staking_rewards_test.go]: same here

File: pkg/participation/participation_manager.go (hunk 1 above)
Comment (muXxer) [reply pkg/participation/participation_manager.go:192]: Because its a map.

**Review by alexsporn — APPROVED** [review]

**Pull request merged**